src/ngx_http_lua_util.h

```diff
@@ -260,6 +260,21 @@ void ngx_http_lua_set_sa_restart(ngx_log_t *log);
     }
 
 
+static ngx_inline size_t
+ngx_http_lua_safe_header_value_len(u_char *str, size_t len)
+{
+    size_t i;
+
+    for (i = 0; i < len; i++, str++) {
+        if (*str == '\r' || *str == '\n') {
```

Comment:
Should we maybe use the `LF` and `CR` macros here instead?

Reply:
Will merge without this suggestion for now if we think it is more readable. We can always update later if we change our mind.

Reply:
Oh yeah, CR and LF are better.